language/tools/move-cli/src/lib.rs

```diff
@@ -311,16 +304,17 @@ impl OnDiskStateView {
         if !path.exists() {
             fs::create_dir_all(path.parent().unwrap())?
         }
-        let mut f = fs::File::create(path)?;
-        Ok(f.write_all(&module_bytes)?)
+
+        Ok(fs::write(path, &module_bytes)?)
```

Comment:
Will this add the `.mv` extension (I guess it will once it rebased on your other PR)?